<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1308</width>
    <height>384</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Vehicle Telemtry and Diagnostics</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="animated">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>1210</x>
      <y>295</y>
      <width>71</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Radar</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>295</y>
      <width>101</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Battery Temp</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SaveLoggerName">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>318</y>
      <width>88</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="LoggerFileName">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>315</y>
      <width>401</width>
      <height>31</height>
     </rect>
    </property>
    <property name="plainText">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Log File Name</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>1080</x>
      <y>39</y>
      <width>211</width>
      <height>210</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="_Temp"/>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>4</y>
      <width>341</width>
      <height>281</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="_SpeedGauge"/>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>1170</x>
      <y>325</y>
      <width>71</width>
      <height>19</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>C</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>4</y>
      <width>341</width>
      <height>281</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="_RPM"/>
   </widget>
   <widget class="QLCDNumber" name="Radar">
    <property name="geometry">
     <rect>
      <x>1190</x>
      <y>315</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>1290</x>
      <y>325</y>
      <width>71</width>
      <height>19</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>M</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_5">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>44</y>
      <width>211</width>
      <height>201</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="_Bat_Perc"/>
   </widget>
   <widget class="QLCDNumber" name="BatteryTemp">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>315</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
